projects/components/src/input/input.component.ts

```diff
@@ -57,6 +59,12 @@ export class InputComponent<T extends string | number> implements ControlValueAc
   @Input()
   public disabled: boolean = false;
 
+  @Input()
+  public min?: number | undefined;
+
+  @Input()
+  public max?: number | undefined;
```

Comment:
Nitpick: I think you could choose one of the `?` or `undefined`.

Reply:
Yeah, only one or the other _should_ be required. We traditionally use just `?`. However, for some reason the IDE linter is complaining and not acknowledging that the `?` is the same as `undefined`. I hate to modify code just for an IDE bug, but seemed fairly harmless just to also add the `| undefined` here.